app-editors/vis/vis-0.4.ebuild

```diff
@@ -25,7 +25,7 @@ RDEPEND="${DEPEND}
 src_prepare() {
 	if use test; then
 		rm -r test || die
-		mv "${WORKDIR}/${MY_PT}" test || die
+		mv "${WORKDIR}/vis-test-${MY_PT}" test || die
```

Comment:
This is still wrong. Also please squash this into the first commit, as it is a breakage introduced in it and there's no point adding the breakage in the first place.